### PR TITLE
Add new bot - lambda_detach_external_layer

### PR DESCRIPTION
**Pull request by TinaSobol**

### CODE REVIEW

File: bots/lambda_detach_external_layer.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+Limitations: The bot will stop running if the proper 'UpdateFunctionConfiguration20150331v2' event is not found.
+'''
+
+import boto3
+import bots_utils
+import botocore.exceptions
+import json
+
+LAYER_NAME_PLACEMENT = 6
+EVENT_NAME = 'UpdateFunctionConfiguration20150331v2'
```
Comment (doronzil) [inline bots/lambda_detach_external_layer.py:15]: Is it the only event relevant for updating the lambda?

File: bots/lambda_detach_external_layer.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+
+def detach_layer_from_lambda(boto_session, lambda_function_name, layer_name_to_delete):
+    lambda_client = boto_session.client('lambda')
+    # Getting lambda's current layers
+    try:
+        current_layers = lambda_client.get_function(FunctionName=lambda_function_name)['Configuration']['Layers']
+    except KeyError:
+        return f'Error: the lambda function ({lambda_function_name}) has no layers attached to it'
+    except botocore.exceptions.ClientError as e:
+        return 'Client error: %s \n' % e
```
Comment (doronzil) [inline bots/lambda_detach_external_layer.py:46]: use f' print format

File: bots/lambda_detach_external_layer.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+    # Remove the layer specified in the event from the layers list
+    try:
+        layer_item_to_delete = next((layer for layer in current_layers if layer_name_to_delete in layer.get('Arn')))
+    except StopIteration:
+        return 'Error: Layer to detach doesn\'t exist or was already detached'
+    else:
+        current_layers.remove(layer_item_to_delete)
```
Comment (doronzil) [inline bots/lambda_detach_external_layer.py:54]: what if there is only one layer attached to the lambda?

**Review by doronzil — CHANGES_REQUESTED** [review]

**Review by RavidAtias — APPROVED** [review]

File: bots/lambda_detach_external_layer.py (hunk 1 above)
Comment (TinaSobol) [reply bots/lambda_detach_external_layer.py:15]: Lior told me it's ok to use 'UpdateFunctionConfiguration20150331v2' only

**Review by doronzil — APPROVED** [review]

**Pull request merged**